ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -245,5 +260,12 @@ const queueWorker = {
 };
 
 settings.onload('Livechat_Routing_Method', function() {
-	RoutingManager.getConfig().autoAssignAgent ? queueWorker.start() : queueWorker.stop();
+	const routingSupportsAutoAssign = RoutingManager.getConfig().autoAssignAgent;
+	logger.queue.debug(`Routing method ${ RoutingManager.methodName } supports auto assignment: ${ routingSupportsAutoAssign }. ${
+		routingSupportsAutoAssign
+			? 'Starting'
+			: 'Stopping'
+	} queue`);
+
+	routingSupportsAutoAssign ? queueWorker.start() : queueWorker.stop();
```

Comment:
This queueWorker is running even when omnichannel is disabled and it's generating a lot of useless debug logs every second.